github/apps.go

```diff
@@ -282,3 +299,32 @@ func (s *AppsService) getInstallation(ctx context.Context, url string) (*Install
 
 	return i, resp, nil
 }
+
+// GetReadWriter converts a body interface into an io.ReadWriter object.
+func GetReadWriter(body interface{}) (io.ReadWriter, error) {
```

Comment:
I don't think this or the next function are necessary. Please remove them.

Reply:
They are used in apps_test.go, and I would like to use them in the ghinstallation package in a subsequent pull request as well. It's useful for comparing results in testing, and will be useful in converting installation token options into a ReadWriter to pass to http.NewRequest in ghinstallation.

https://github.com/bradleyfalzon/ghinstallation